implentio/ui-toolkit
# fix: restore default css variables

## Description
Brought back default CSS variables to revert back unexpected UI regression.
They were removed by me thinking the values were unused, but this resulted in some Tailwind utility classes not working (e.g. `rounded-md` by removing `--radius: 0.5rem`)

## Link to JIRA ticket
No ticket for this one, spotted on Storybook site.

## Applicable Screenshots
<img width="163" alt="Screenshot 2024-02-05 at 4 29 23 AM" src="https://github.com/implentio/ui-toolkit/assets/155646835/581362f6-a109-4dd9-9163-cd53d66eb181">
<img width="158" alt="Screenshot 2024-02-05 at 4 29 55 AM" src="https://github.com/implentio/ui-toolkit/assets/155646835/7c449663-3299-424b-95e0-12aff0b8fb35">

## Testing Instructions
Outline the steps required to test your changes effectively (Replace the following to what makes sense for your PR):
- Fetch and checkout my branch `git fetch && git checkout fix/restore-default-css-variables)`
- Run `npm run storybook`
- Check `StatusChip` component to see they're rounded.

## Checklist:
Check what is applicable by putting an 'x' in between the []:
